flink-shaded-netty-4/pom.xml

```diff
@@ -94,12 +94,12 @@ under the License.
                                 <echo message="renaming native epoll library" />
                                 <move todir="${project.build.directory}/unpacked/META-INF/native" includeemptydirs="false">
                                     <fileset dir="${project.build.directory}/unpacked/META-INF/native"/>
-                                    <mapper type="glob" from="libnetty_transport_native_epoll_x86_64.so" to="liborg_apache_flink_shaded_netty4_netty_transport_native_epoll_x86_64.so"/>
+                                    <mapper type="glob" from="libnetty_transport_native_epoll_*.so" to="liborg_apache_flink_shaded_netty4_netty_transport_native_epoll_*.so"/>
                                 </move>
                                 <echo message="renaming native kqueue library" />
                                 <move todir="${project.build.directory}/unpacked/META-INF/native" includeemptydirs="false">
                                     <fileset dir="${project.build.directory}/unpacked/META-INF/native"/>
-                                    <mapper type="glob" from="libnetty_transport_native_kqueue_x86_64.jnilib" to="liborg_apache_flink_shaded_netty4_netty_transport_native_kqueue_x86_64.jnilib"/>
+                                    <mapper type="glob" from="libnetty_transport_native_kqueue_*.jnilib" to="liborg_apache_flink_shaded_netty4_netty_transport_native_kqueue_*.jnilib"/>
```

Comment:
if these patterns actually work we could merge the epoll/kqueue moves and significantly streamline things.

```
from:  libnetty*
to:    liborg_apache_flink_shaded_netty*
```

Reply:
Not sure if there is other stuff in the native directory that we wouldnt want relocated.